sr_ronex_launch/launch/passthrough_controllers.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+  <node name="ronex_ctrl_spawner"
```

Comment:
shouldn't it be included in sr_ronex.launch (to remove the duplication)? I would also move it to the `loaders` folder but happy if you want to keep it in launch too.